packages/amplify_authenticator/test/ui/tab_view_test.dart

```diff
@@ -143,6 +143,7 @@ void main() {
   group('AuthenticatorTabView', () {
     setUp(() {
       binding.window.clearPlatformBrightnessTestValue();
+      binding.window.physicalSizeTestValue = const Size(2400, 2400);
```

Comment:
This had ti be adjusted for the Amplify theme tests because the authenticator is now bigger than the screenshot. I updated for all the others as well.